packages/cli/src/commands/deploy.ts

```diff
@@ -63,8 +64,47 @@ async function run(options: Required<DeployCommandOptions>): Promise<void> {
   const logger = createLogger({ verbose, quiet, logFile });
 
   const deployer = await Deployer.create(manifestFile, logger);
+
+  // set primary job before running
+  let primaryJobName = deployer.manifest.primaryJobName;
+  const jobNames = Object.keys(deployer.manifest.jobs);
+  if (primaryJobName) {
+    // validate primary job name
+    if (!jobNames.find((jobName) => jobName === primaryJobName)) {
+      logger.error(intlMsg.commands_deploy_error_primaryJobNotFound());
+      process.exit(1);
+    }
+  } else {
+    // default to first job
+    primaryJobName = jobNames[0];
+  }
+
   const jobResults = await deployer.run();
 
+  // get deployment uri
+  const primaryJob = jobResults.find(
+    (job) => job.name === primaryJobName
+  ) as DeployJobResult;
+  const deploymentStep = primaryJob.steps.length - 1;
+  const deploymentUri = primaryJob.steps[deploymentStep].result.uri;
+
+  // write deployment uri to file
+  const manifestDir = path.dirname(manifestFile);
+  const deploymentFilepath = path.join(manifestDir, "polywrap.deployment.txt");
```

Comment:
I'm not a fan of using the `polywrap` name here, I think we should reserve the `polywrap` file name for toolchain configuration files. Similarly, we reserve the `wrap` file name for wrap standard files.

This file to me is super similar to GitHub page's "CNAME" file, which redirects to the external domain the page is being hosted at. Here we're redirecting to an external URI where the package is being hosted at.

It's for this reason that I'd be preferable to both the names `WRAP` and `URI`, where they're simple text files containing 1 line of text, which is the URI to be used for wrapper resolution.

Thoughts?

Reply:
I renamed it to `URI.txt`

Reply:
and yeah, that all makes sense and sounds good!

Reply:
let me know if you actually preferred `WRAP.txt`. It's an easy change to make now, but later it will be harder if we start using the github resolver